air/src/options.rs

```diff
@@ -82,6 +94,8 @@ pub struct ProofOptions {
     field_extension: FieldExtension,
     fri_folding_factor: u8,
     fri_remainder_max_degree: u8,
+    num_partitions: u8,
+    min_partition_size: u8,
```

Comment:
is there a reason not to store these as a single `partition: PartitionOptions` field? Then, these can be exposed from options as a single field.

Reply:
Makes sense, changed